### PR TITLE
Issue 2: updated dependencies.

### LINKED ISSUE #2: Update core dependencies
Update `react`, `react-dom` and `react-scripts` to the latest versions

### PR DESCRIPTION
**Pull request by ekostandov**

### CODE REVIEW

File: my-app/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+/node_modules
```
Comment (singvarr) [inline my-app/.gitignore]: Let's move all project to root level  from `my-app`

File: my-app/package.json [hunk 2]
```diff
@@ -3,19 +3,20 @@
...
   "dependencies": {
-    "lodash": "^4.17.5",
-    "prop-types": "^15.6.1",
-    "react": "^16.2.0",
-    "react-dom": "^16.2.0",
-    "react-redux": "^5.0.7",
-    "react-router-dom": "^4.2.2",
-    "react-scripts": "1.1.1",
+    "lodash": "^4.17.21",
+    "prop-types": "^15.7.2",
```
Comment (singvarr) [inline my-app/package.json]: You don't need to install `prop-types` manually. It's loaded with `react` after v.16

File: my-app/package.json [hunk 3]
```diff
@@ -3,19 +3,20 @@
...
+    "react": "^16.14.0",
```
Comment (singvarr) [inline my-app/package.json]: Let's try to install the latest versions of `react` and `react-dom`

**Review by singvarr — CHANGES_REQUESTED** [review]

File: my-app/package.json (hunk 3 above)
Comment (ekostandov) [reply my-app/package.json]: All changes done, accordingly to the comments.

**Review by singvarr — APPROVED** [review]

**Pull request merged**